broker-daemon/block-order-worker/index.js

```diff
@@ -100,6 +100,41 @@ class BlockOrderWorker extends EventEmitter {
     }
 
     const blockOrder = new BlockOrder({ id, marketName, side, amount, price, timeInForce })
+    const baseEngine = this.engines.get(blockOrder.baseSymbol)
+    const counterEngine = this.engines.get(blockOrder.counterSymbol)
+    const { address: counterSymbolAddress } = await this.relayer.paymentChannelNetworkService.getAddress({symbol: blockOrder.counterSymbol})
+    const { address: baseSymbolAddress } = await this.relayer.paymentChannelNetworkService.getAddress({symbol: blockOrder.baseSymbol})
+
+    if (blockOrder.isBid) {
```

Comment:
Have you considered ways to consolidate this with the `isAsk` block below (without sacrificing readability)?

Reply:
I would prefer to clean this up in the next PR when I combine with the earmarking. Right now the code is very readable and I would like to leave it but I will condense in the next PR if thats cool with you.